### PR TITLE
Adding Gridsome

### PR DESCRIPTION
**Pull request by NickStees**
Gridsome a Vue powered static site generator. 

- [gridsome.org](https://gridsome.org/)
- [github.com/gridsome/gridsome](https://github.com/gridsome/gridsome)
- [www.staticgen.com/gridsome](https://www.staticgen.com/gridsome)

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -4123,6 +4123,16 @@
       },
       "website": "http://www.greenvalley.nl/Public/Producten/Content_Management/CMS"
     },
+    "Gridsome": {
+      "cats": [
+        57
+      ],
+      "icon": "Gridsome.svg",
+      "meta": {
+        "generator": "Gridsome (v[\\d.]+)?\\;version:\\1"
```
Comment (jvoisin) [inline src/apps.json]: What about `"generator": "^Gridsome v([\\d.]+)$\\;version:\\1"` instead?

Comment (NickStees) [reply src/apps.json]: Sounds good to me

Comment (NickStees) [reply src/apps.json]: @jvoisin I am not sure how to add Vue.js as a dependency, I found something about `implies` so I am going to assume thats what you meant.

Also FYI when researching how to add Gridsome, I noticed the Docs have a broken link. 
> Refer to the specification for more detail. https://www.wappalyzer.com/docs#specification

File: src/apps.json [hunk 2]
```diff
@@ -4123,6 +4123,17 @@
...
       "website": "http://www.greenvalley.nl/Public/Producten/Content_Management/CMS"
     },
+    "Gridsome": {
+      "cats": [
+        57
+      ],
+      "icon": "Gridsome.svg",
+      "implies": "Vue.js",
+      "meta": {
+        "generator": "^Gridsome (v[\\d.]+)?\\;version:\\1"
```
Comment (jvoisin) [inline src/apps.json]: Why the trailing `?` instead of a `$`?

File: src/apps.json (hunk 1 above)
Comment (jvoisin) [reply src/apps.json]: Yup, my bad.

File: src/apps.json (hunk 2 above)
Comment (NickStees) [reply src/apps.json]: @jvoisin Sorry need to come clean, I knew the Jekyll meta tag as pretty much identical, so I copy and pasted the Jekyll one, and just modified it. I actually can barely tolerate Regex, my mind numbs up every time  :) 

It needs to match something like `<meta data-vue-tag="true" name="generator" content="Gridsome v0.5.2"/>` and the Jekyll one looked very similar. `<meta name="generator" content="Jekyll v3.7.4" />` so I copy/pasted.

I am going to assume you are better at Regex ;) and push exactly what you suggested. 

FYI the Jekyll Regex may need fixed, its currently `"generator": "Jekyll (v[\\d.]+)?\\;version:\\1"`

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**